media-libs/opensubdiv/opensubdiv-3.3.0.recipe

```diff
@@ -22,11 +22,16 @@ PROVIDES="
 	cmd:far_regression = $portVersion
 	cmd:hbr_baseline = $portVersion
 	cmd:hbr_regression = $portVersion
+	cmd:stringify = $portVersion
 	lib:libosdCPU$secondaryArchSuffix = 3.3.0 compat >= 3
+	lib:libosdGPU$secondaryArchSuffix = 3.3.0 compat >= 3
```

Comment:
You might wish to define libVerson and libVersonCompat before PROVIDES:
```diff
 SECONDARY_ARCHITECTURES="?x86"
 
+libVerson="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
```
and then use them here, in PROVIDES, as well as in PROVIDES_devel and in defineDebugInfoPackage:
```diff
-	lib:libosdCPU$secondaryArchSuffix = 3.3.0 compat >= 3
+	lib:libosdCPU$secondaryArchSuffix = $libVersionCompat
-	lib:libosdGPU$secondaryArchSuffix = 3.3.0 compat >= 3
+	lib:libosdGPU$secondaryArchSuffix = $libVersionCompat
```